server/filestore.go

```diff
@@ -971,6 +971,7 @@ func (mb *msgBlock) spinUpFlushLoop() {
 		mb.mu.Unlock()
 		return
 	}
+	mb.flusher = true
```

Comment:
May not need the `setInFlusher()` function and its call in flushLoop() anymore?

Reply:
I looked at that but liked the symmetry there so left it. Can remove it and put comment there if you feel strongly about it.

Reply:
that's fine.